python/paddle/fluid/tests/unittests/test_recurrent_op.py

```diff
@@ -602,5 +607,76 @@ def dot_attention(query, memory):
         return rnn()
 
 
+class RecurrentOpTest3(RecurrentOpTest1):
```

Comment:
I guess you were copying the old test class name. In fact, var1, var2, class1, class2 are very bad class names ...... you should do the best from new code.

You can write "RecurrentOpStopGradientTest"

Reply:
Done.